var/spack/repos/builtin/packages/qgraf/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Qgraf(Package):
+    """Qgraf is a computer program that generates Feynman diagrams for various types of QFT models"""
+
+    homepage = "http://cfif.ist.utl.pt/~paulo/qgraf.html"
+    url      = "http://anonymous:anonymous@qgraf.tecnico.ulisboa.pt/v3.4/qgraf-3.4.2.tgz"
+
+    version('3.4.2', sha256='cfc029fb871c78943865ef8b51ebcd3cd4428448b8816714b049669dfdeab8aa')
+
+
+    def install(self, spec, prefix):
+        fortran = Executable(self.compiler.fc)
+        fortran('qgraf-{0}.f'.format(self.spec.version), '-o', 'qgraf')
+        install('*', prefix)
```

Comment:
```suggestion
        install_tree('.', prefix)
```
This is more robust in the case of directories.